Makefile

```diff
@@ -41,7 +41,7 @@ install: manifests
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 .PHONY: deploy
 deploy: manifests docker-build
-	kubectl apply -f config/crds
+	kubectl apply -f config/crds --validate=false
```

Comment:
is there a reason you turned off validation?

Reply:
I'm having problems with this for all our CRDs when I try to deploy them on a 3.11 oc cluster up, it doesn't like the status fields, complains about missing required fields in status. Will remove this for now as I wasn't able to get it working anyhow.